tests/test_filters.py

```diff
@@ -527,6 +524,32 @@ def test_glyph():
         sphere.glyph(geom=geoms, indices=indices[:-1])
 
 
+def test_glyph_cell_point_data():
+    sphere = pyvista.Sphere(radius=3.14, theta_resolution=5, phi_resolution=5)
```

Comment:
Do we need exact reproducibility of the swirly example for the test? Or can we use the sphere fixture?

Reply:
I agree.  I simplified in both glyph tests using `pyvsta.Sphere()`.  Is there an advantage to the fixture?

Reply:
If we're creating the mesh ourselves anyway then probably not. But it does remove some logic from the test, which is good. If nothing else it makes the test more readable by using the same fixtures.

Reply:
it is a bit of a wash in terms of line count needed.  It does make it more explicit that "Normals" are required for part of the test, which is nice.

Reply:
I didn't remember that the fixture was missing the normals, I thought it was the other way around.